### DISCUSSION TITLE
Sugguestion On Windows Development.

### DISCUSSION THREAD

**Discussion by sanad89d** [NickvisionApps/Parabolic#290] [General]
Just a sugguestion, Dont stop windows development, Just don't publish it on microsoft store, user that need the software can come and get it from github.

**Comment by fsobolev** [comment]
As mentioned in [the article](https://nickvision.org/2023/05/23/removing-windows-support.html):

> Being built with the new [Windows App SDK](https://github.com/microsoft/WindowsAppSDK), Microsoft basically forces developers into using MSIX packaging instead of EXEs and like I said, we didn’t mind, however, this came with a ton of issues:
> 
> First, MSIX packages cannot be installed freely as EXEs can. They require a digital signature which can only be generated using complicated CLI tools from Microsoft. Although increasing security, there are no options for power-users who want to quickly test their unsigned MSIX packages.

Currently the plan is to make apps work on Windows with GNOME UI . I have some progress on this, so hopefully I will make everything that is needed before next releases (at least for Denaro) and Windows builds will most likely be distributed using [Scoop](https://scoop.sh).

**Reply by nlogozzo** [reply]
As @fsobolev mentioned from the article, with the frameworks we were using for Windows, 1, we couldn't easily distribute our MSIX packages and, 2, our developers were forced to use Windows to develop and test the WinUI version and literally all of our developers use Linux, so this didn't really work.

As he said, we are working to get the GNOME UI of our apps working natively on Windows for our Windows users to use :)
